src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendOrderContestedAboutToSubmitHandler.java

```diff
@@ -59,9 +61,13 @@ public AboutToStartOrSubmitCallbackResponse handle(CallbackRequest callbackReque
 
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
 
-        printAndMailGeneralOrderToParties(caseDetails, userAuthorisation);
-        printAndMailHearingDocuments(caseDetails, userAuthorisation);
-        stampFinalOrder(caseDetails, userAuthorisation);
+        try {
+            printAndMailGeneralOrderToParties(caseDetails, userAuthorisation);
+            printAndMailHearingDocuments(caseDetails, userAuthorisation);
+            stampFinalOrder(caseDetails, userAuthorisation);
+        } catch (InvalidCaseDataException e) {
+            return AboutToStartOrSubmitCallbackResponse.builder().errors(ImmutableList.of(e.getMessage())).build();
```

Comment:
do we need to caseData also in reponse?

Reply:
No, if error then we don't require casedata here